templates/default/authoritative_service.conf.erb

```diff
@@ -0,0 +1,31 @@
+<% if @config_dir %>
```

Comment:
You may want to consider putting in comments for this file if the config format supports commented lines to note that the file is managed by chef and to not manually edit it.